lib/components/user/monitored-trip/trip-notifications-pane.js

```diff
@@ -44,64 +102,66 @@ class TripNotificationsPane extends Component {
-            your account settings once the trip has been saved.
-            </HelpBlock>
-          </FormGroup>
-
-          <FormGroup>
-            <ControlLabel>When would you like to receive notifications about delays or disruptions to your trip?</ControlLabel>
-
-            <FormGroup>
-              <ControlLabel>Check for delays or disruptions:</ControlLabel>
-              <Field
-                as={FormControl}
-                componentClass='select'
-                name='leadTimeInMinutes'
+                <option value>Yes (default)</option>
+                <option value={false}>No</option>
+              </Select>
+            </li>
+            <li>
+              <Select
+                label='There are delays or disruptions of more than'
+                name='departureVarianceMinutesThreshold'
               >
-                <option value={15}>15 min. prior</option>
-                <option value={30}>30 min. prior (default)</option>
-                <option value={45}>45 min. prior</option>
-                <option value={60}>60 min. prior</option>
-              </Field>
-            </FormGroup>
-            <Alert bsStyle='warning'>
-                    Under construction!
-              <FormGroup>
-                <ControlLabel>Notify me if:</ControlLabel>
-                <Checkbox>A different route or transfer point is recommended</Checkbox>
-                <Checkbox>There is an alert for a route or stop that is part of my journey</Checkbox>
+                <option value={5}>5 min (default)</option>
+                <option value={10}>10 min</option>
+                <option value={15}>15 min</option>
+              </Select>
+            </li>
+          </SettingsListWithAlign>
 
-                      Your arrival or departure time changes by more than:
-                <FormControl componentClass='select' defaultValue={5} placeholder='select'>
-                  <option value={5}>5 min. (default)</option>
-                  <option value={10}>10 min.</option>
-                  <option value={15}>15 min.</option>
-                </FormControl>
-              </FormGroup>
-            </Alert>
-          </FormGroup>
+          <h4>
+            <SettingsToggle
+              aria-expanded={showAdvancedSettings}
+              aria-label='Toggle advanced settings'
+              onClick={this._handleToggleAdvancedSettings}
+              type='button'
+            >
+              <Icon type={showAdvancedSettings ? 'caret-down' : 'caret-right'} />
+              Advanced settings
+            </SettingsToggle>
+          </h4>
+          {showAdvancedSettings && (
+            <SettingsList>
+              <li>
+                <Select
+                  FieldComponent={InlineField}
+                  label='Monitor this trip'
+                  name='leadTimeInMinutes'
+                >
+                  <option value={15}>15 min</option>
+                  <option value={30}>30 min (default)</option>
```

Comment:
Because it's not the first option, 30 minutes is not selected by default. For this select and the others, we may want to add a selected prop? I'm not too familiar with how Formik handles `<select/>` inputs though, so I'm open to alternative suggestions.

Reply:
Forgot about that, glad you pointed it out.

Reply:
That was tricker. The default value was already populated correctly in the trip object, but the I passed the wrong component for styling, so it's fixed now in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/4801a00160dbc1d75bb375d39d2cbb0bb4b4c34e.